system_strings/sysmes_text.en

```diff
@@ -1111,20 +1111,20 @@ More vampire murders have been^reported. Is this case not yet^over...?
 Vlov is supposed to be dead.^Only one other vampire remains.^I must find Arcueid.
 I'm back at the mansion. If^I calm down and organize my^thoughts, I may be able to^remember something.
 The first time I met her was^in passing. If I can't rely on^chance, I'll have to rely on my^intuition and search for her on^foot.
-I can't keep treating her like^a child forever. If I talk to^her, we should be able to come^to an understanding. Rather, I^should make a bold entrance.
-Her ability to make even adults^cower in fear is amazing. Though^she may seem calm, the family^head is still quite immature.
-I feel a strange atmosphere^emanating from the living room.^Hisui appears, and woodenly^informs me that I have a^visitor...
+I can't keep treating her like^a little kid forever. If I^talk to her, we should be able to^come to an understanding. Rather, I^should make a bold entrance.
+Her ability to make even adults^cower in fear is amazing. Though^she may seem calm, the family^head is still really immature.
+I feel a strange atmosphere^emanating from the living room.^Hisui appears, and stoically^informs me that I have a^visitor...
 I've made my desperate escape^from the mansion. I shudder to^think what's in store for me^later, but I need to talk to^this woman now.
 I question Arcueid about the^vampire incidents. The truth^is much too simple.
-Arcueid, the hunter of Undead.^I'm once again reminded of the^condition of the city, which has^been contaminated by the Dead...
-I question what I, as a human,^can do. However, there's no^getting out of it now, so I^offer to help Arcueid――
-We're starting our nighttime^investigation. I accept Arcueid's^proposal and head out into the^city without my glasses on.
-I'll talk to her a little more.^Now that I've decided to help^her exterminate vampires, there's^too much I still don't know.
+Arcueid hunts the Undead.^I am once again struck by the state^of the city, infected by^the Dead Apostles...
+I know a human shouldn't think like^this, but I can't help but offer to^cooperate with Arcueid.———
```

Comment:
What's with the `.―`

Reply:
typo i think

Reply:
it is, my bad

Reply:
can't exactly let you take the blame for things that i wrote :aokoheh:.